pkgs/tools/misc/kalibrate-hackrf/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, fftw, hackrf, libusb1 }:
+
+stdenv.mkDerivation rec {
+  name = "kalibrate-hackrf-20160827";
+
+  # There are no tags/releases, so use the latest commit from git master.
+  # Currently, the latest commit is from 2016-07-03.
+  src = fetchFromGitHub {
+    owner = "scateu";
+    repo = "kalibrate-hackrf";
+    rev = "2492c20822ca6a49dce97967caf394b1d4b2c43e";
+    sha256 = "1jvn1qx7csgycxpx1k804sm9gk5a0c65z9gh8ybp9awq3pziv0nx";
+  };
+
+
+
+  buildInputs = [ autoreconfHook pkgconfig fftw hackrf libusb1 ];
```

Comment:
`autoreconfHook` and `pkgconfig` belong in `nativeBuildInputs` as they are run on the architecture of the build host (native)

Also you might get rid of two of the newlines :wink: